src/common/transformations/include/transformations/transpose_sinking/ts_strided_slice.hpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2022-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/pass/graph_rewrite.hpp"
+#include "openvino/pass/pass.hpp"
+#include "transformations/transpose_sinking/ts_base.hpp"
+#include "transformations_visibility.hpp"
+
+namespace ov {
+namespace pass {
+namespace transpose_sinking {
+
+class TRANSFORMATIONS_API TSStridedSliceForward;
+class TRANSFORMATIONS_API TSStridedSliceBackward;
+
+}  // namespace transpose_sinking
+}  // namespace pass
+}  // namespace ov
+
```

Comment:
Could you please add some common comments about transformations.

Reply:
done